compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -810,6 +812,7 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
             };
             v.visit_block(block);
             if let Some(expr) = v.expr && let Call(_, [expr]) = expr.kind {
+                debug!("anan expr: {:#?}", expr);
```

Comment:
anan?

Reply:
ah, forgot to remove my debug code.
clean it.